src/qt/rpcconsole.cpp

```diff
@@ -8,6 +8,7 @@
 #include "clientmodel.h"
 #include "guiutil.h"
 #include "peertablemodel.h"
+#include "bantablemodel.h"
```

Comment:
Not even a nit, but a please: Can you add #include "bantablemodel.h" above clientmodel :)?